PKGBUILD

```
@@ -59,12 +59,12 @@ package() {
     "$pkgdir/opt/${pkgname}/chrome-wrapper"
   # Install proper desktop file
   install -Dm644 "${srcdir}/${_archdir}/helium.desktop" \
-    "${pkgdir}/usr/share/applications/${_binaryname}.desktop"
+    "${pkgdir}/usr/share/applications/${_pkgname}.desktop"
   # Install icon for desktop file
   install -Dm644 "${pkgdir}/opt/${pkgname}/product_logo_256.png" \
-    "${pkgdir}/usr/share/pixmaps/${_binaryname}.png"
+    "${pkgdir}/usr/share/pixmaps/${_pkgname}.png"
   install -Dm644 "${pkgdir}/opt/${pkgname}/product_logo_256.png" \
-    "${pkgdir}/usr/share/icons/hicolor/256x256/apps/${_binaryname}.png"
+    "${pkgdir}/usr/share/icons/hicolor/256x256/apps/${_pkgname}.png"
```

```suggestion
    "${pkgdir}/usr/share/pixmaps/${_binaryname}.png"
```
Icon names should match `Name=` defined by the `.desktop` entry, i.e. `helium-browser`, to insure desktop runners etc. resolve the icon correctly.